servers.json

```diff
@@ -1059,11 +1059,18 @@
             "alpha.rip"
         ]
     },
-           {
+       {
```

Comment:
This is still the incorrect bracket formatting

Reply:
Ready